apps/web/src/services/dashboards/widgets/_helpers/widget-schema-helper.ts

```diff
@@ -90,5 +90,12 @@ export const getRefinedSchemaProperties = (
     widgetOptions?: WidgetOptions,
 ): string[] => {
     const optionExistProperties = storedProperties.filter((property) => !!get(widgetOptions, property));
+
+    // Change deprecated name from order
+    const index = optionExistProperties.indexOf('asset_query_set');
+    if (index > -1) {
+        optionExistProperties.splice(index, 1, WIDGET_OPTIONS_SCHEMA_PROPERTIES.cloud_service_query_set.key);
+    }
+
```

Comment:
suggestion: I think it would be better to manage the codes to locate all conversion codes at dashboard-detail-info-store.